src/webVitals.ts

```diff
@@ -12,24 +13,30 @@ export function reportWebVitals(metric: NextWebVitalsMetric) {
   collectedMetrics.push({ route: window.__NEXT_DATA__?.page, ...metric });
   // if Axiom env vars are not set, do nothing,
   // otherwise devs will get errors on dev environments
-  if (!isEnvVarsSet) {
+  if (!config.isEnvVarsSet()) {
     return;
   }
   throttledSendMetrics();
 }
 
 function sendMetrics() {
-  const body = JSON.stringify({
-    webVitals: collectedMetrics,
-    environment: vercelEnv,
-  });
+  const body = JSON.stringify(config.wrapWebVitalsObject(collectedMetrics));
+  const headers = {
+    Authorization: `Bearer ${config.token}`,
+    type: 'application/json',
+  };
```

Comment:
Can we only add the auth header if config.token ≠ undefined?

Reply:
done